src/js/pages/TwitterHandleLanding.jsx

```diff
@@ -22,7 +22,7 @@ const DelayedLoad = React.lazy(() => import(/* webpackChunkName: 'DelayedLoad' *
 
 // The component that gets called for paths like https://localhost:3000/sierraclub
 // A Twitter handle is the username that appears at the end of your unique Twitter URL
-export default class TwitterHandleLanding extends Component {
+export default class  TwitterHandleLanding extends Component {
```

Comment:
This can be done in a future pull request, but I notice there is an extra space before "TwitterHandleLanding" which should be removed.